src/Data/CurrentUser/errorToast.tsx

```diff
@@ -0,0 +1,23 @@
+import { toast } from 'react-toastify'
+import { ClientError } from 'scrivito'
+
+export function errorToast(title: string, error: unknown): void {
+  toast.error(
+    <>
+      <div>{title}</div>
+      {error instanceof ClientError ? (
+        <>
+          <div className="text-small">{error.message}</div>
```

Comment:
* [ ] FYI `<small>{error.message}</small>` also looks good and would even be more sematic.